test/e2e/eraser_test.go

```diff
@@ -86,6 +91,7 @@ func TestRemoveImagesFromAllNodes(t *testing.T) {
 			}
 
 			// deploy imageJob config
+			time.Sleep(45 * time.Second)
```

Comment:
why are we sleeping here? can we watch instead of sleep?

Reply:
I just made a comment on #134 to this effect. We are sleeping because the `wait.For` doesn't actually result in the right state for deletion. The images still appear to eraser to be `running`. We will have to ditch the `wait.For` calls in favor of something else. I only used `time.Sleep` to verify that the issue was a race condition of some kind. How would you recommend checking that the pods are absolutely done?

Reply:
- [ ] Determine a better way to watch for pod & deployment deletion before deploying eraser in tests.